packages/ui/ToggleItem/ToggleItem.js

```diff
@@ -136,6 +162,31 @@ const ToggleItemBase = kind({
 		 */
 		iconPosition: PropTypes.oneOf(['before', 'after']),
 
+		/**
+		 * An additional customizable icon component with more granular positioning rules. This
+		 * should only be used *after* specifying the `icon` property, as the positioning for this
+		 * offers the ability to place this in front of or behind the existing `icon`.
+		 * See `itemIconPosition` for options.
+		 *
+		 * @type {Node}
+		 * @public
+		 */
+		itemIcon: PropTypes.node,
+
+		/**
+		 * Specifies on which position (`'before'`, `'beforeText'`, `'afterText'`, `'after'`) of the
+		 * text the `itemIcon` appears.
+		 *
+		 * * before - first element in the item
+		 * * beforeText - if a "before" `icon` is present, this will be between the icon and the text
```

Comment:
`ToggleItem` makes no restriction on its children so `'beforeText'` is a tad misleading.  Perhaps `'beforeChildren'`?

Reply:
ok.